src/components/try-hcp-callout/content.ts

```diff
@@ -59,4 +59,11 @@ export const tryHcpCalloutContent: Record<
 		ctaText: 'Try cloud for free',
 		ctaUrl: 'https://portal.cloud.hashicorp.com/sign-up',
 	},
+	vagrant: {
+		heading: 'Vagrant Cloud',
+		description:
```

Comment:
It seems like the description isn't used by the prebuilt callout anymore, should we remove it from the interface?

Reply:
nvm, it was fixed in #1421